s3tests_boto3/functional/__init__.py

```diff
@@ -127,9 +127,6 @@ def nuke_prefixed_buckets(prefix, client=None):
             try:
                 response = client.delete_bucket(Bucket=bucket_name)
             except ClientError:
-                # if DELETE times out, the retry may see NoSuchBucket
-                if response['Error']['Code'] != 'NoSuchBucket':
-                    raise ClientError
```

Comment:
What if you save the first or last client error and re-raise it after the loop completes?

Reply:
For example, 
```
error A occurs when delete bucket-1
...
error B occurs when delete bucket-2
```
If we save the last client error and re-raise it after the loop completes, user knows there is an exception B. then he debugs and make some effort to resolve it.
then he have to run s3tests again to know about exception A. 
Maybe this path is a little time-cost or inefficiency.

Can we print every err like this:
```
try:
    response = client.delete_bucket(Bucket=bucket_name)
except ClientError:
    print response
    pass
```

Reply:
If the error is not raised then no one will look at the logs...

Reply:
> If the error is not raised then no one will look at the logs...

Agree. I would save the last client error and re-raise it after the loop completes.